behaviour/graql/language/match.feature

```diff
@@ -550,6 +561,75 @@ Feature: Graql Match Clause
       """
 
 
+  Scenario: when matching by a type whose label doesn't exist, an error is thrown
+    Then graql get throws
+      """
+      match $x isa ganesh; get;
+      """
+    Then the integrity is validated
+
+
+  Scenario: when matching by a type id that doesn't exist, an empty result is returned
+    When get answers of graql query
+      """
+      match $x isa $type; $type id V1337; get;
+      """
+    Then answer size is: 0
+
+
+  Scenario: when matching by a relation type whose label doesn't exist, an error is thrown
+    Then graql get throws
+      """
+      match ($x, $y) isa $type; $type type jakas-relacja; get;
+      """
+    Then the integrity is validated
+
+
+  Scenario: when matching a non-existent type label to a variable from a generic `isa` query, an error is thrown
+    Then graql get throws
+      """
+      match $x isa $type; $type type polok; get;
+      """
+    Then the integrity is validated
+
+
+  Scenario: matching that the same variable is of two types is allowed
```

Comment:
...but returns no answers (if reasoning is off :P) should we include thatbit?

Reply:
Ah yes of course!

Reply:
I think this was one of the first cases I wrote in this category, before I'd settled my mind on the pattern: "when blah, [an error is thrown | an empty result is returned]"

Updated to `when matching that the same variable is of two types, an empty result is returned`